### ISSUE TITLE
ip used

### ISSUE THREAD

**Issue by Caticer** [IPRoyal/pawns-cli#3]
I cant run on my server
`{"happened_at":"2022-01-08T12:47:27Z","name":"not_running","parameters":{"error":"ip_used"}}`

**Comment by alexsorel** [comment]
Are you already running the program on another system, using the same IP? It seems that only one program can run on the same residential IP.

**Comment by Caticer** [comment]
Nope. I dont have another device and program not running. (Ubuntu 20.04 LTS). I can send system logs if you want
